ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__cnv-odf-ocp4.18-lp-interop.yaml

```diff
@@ -64,7 +60,9 @@ tests:
               {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
             ]
         }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
       FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp", "self-managed-lp", "cnv-lp"]'
+      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: lbednar@redhat.com
```

Comment:
I think that for LP job, someone from LP should be the default contact. Unless it's CNV specific.

Reply:
[see conversation in Slack thread](https://redhat-internal.slack.com/archives/C04QDE5TK1C/p1732547000512159?thread_ts=1732307276.267649&cid=C04QDE5TK1C)